sources/packages/forms/src/form-definitions/studentfinancialinformationappeal.json

```diff
@@ -0,0 +1,386 @@
+                    "key": "studentNewTaxReturnIncome",
+                    "placeholder": "",
+                    "prefix": "$",
+                    "suffix": "",
+                    "defaultValue": "",
+                    "protected": false,
+                    "persistent": true,
+                    "hidden": false,
+                    "clearOnHide": true,
+                    "delimiter": true,
+                    "decimalLimit": 0,
+                    "requireDecimal": false,
+                    "validate": {
+                        "required": true,
+                        "multiple": "",
+                        "custom": "valid = (input >= 0 && input < data.maxMoneyValue) ? true : 'The number you have entered is outside the expected range.';"
+                    },
+                    "conditional": {
+                        "show": "",
+                        "when": null,
+                        "eq": ""
+                    },
+                    "type": "currency",
+                    "labelPosition": "top",
+                    "tags": [],
+                    "properties": {},
+                    "redrawOn": "calculatedTaxYear",
+                    "lockKey": true,
+                    "hideLabel": false
+                }
+            ],
+            "type": "panel",
+            "breadcrumb": "default",
+            "hideLabel": true,
+            "tags": [],
+            "conditional": {
+                "show": "",
+                "when": null,
+                "eq": ""
+            },
+            "properties": {}
+        },
+        {
+            "clearOnHide": false,
+            "key": "panel2",
+            "input": false,
+            "title": "Dependant care costs panel",
+            "theme": "default",
+            "tableView": false,
+            "components": [
+                {
+                    "key": "panel2Content",
+                    "label": "Content",
+                    "input": false,
+                    "tag": "h3",
+                    "attrs": [
+                        {
+                            "value": "",
+                            "attr": ""
+                        }
+                    ],
+                    "className": "category-header-medium primary-color",
+                    "content": "Dependant care costs",
+                    "type": "htmlelement",
+                    "hideLabel": true,
+                    "tags": [],
+                    "conditional": {
+                        "show": "",
+                        "when": null,
+                        "eq": ""
+                    },
+                    "properties": {}
+                },
+                {
+                    "autofocus": false,
+                    "input": true,
+                    "tableView": true,
+                    "inputType": "radio",
+                    "label": "<strong>Will you have a child age 11 years or under in daycare? </strong>",
```

Comment:
Using `strong` in labels seems to be a practice in many places but I believe that we should stop doing it since it may cause some impacts in the accessibility.
Other forms defined the same using `"customClass": "label-bold"`.

Reply:
@guru-aot @lewischen-aot @andrepestana-aot during the investigation we figured out that we no longer need the `<strong>` in the label because we have a style already taking care of it `.formio-component .col-form-label`.